interface/ceed-operator.c

```diff
@@ -248,6 +252,50 @@ int CeedCompositeOperatorAddSub(CeedOperator compositeop, CeedOperator subop) {
   return 0;
 }
 
+/**
+  @brief Duplicate a CeedOperator with a refrence Ceed to fallback for advanced
+           CeedOperator functionality
+
+  @param op           CeedOperator to create fallback for
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Developer
+**/
+int CeedOperatorCreateFallback(CeedOperator op) {
+  int ierr;
+
+  // Fallback Ceed
+  const char *resource;
+  ierr = CeedOperatorGetFallbackResource(op, &resource);
+  Ceed ceedref;
+  ierr = CeedInit(resource, &ceedref); CeedChk(ierr);
+  op->fallbackceed = ceedref;
```

Comment:
Can we have one `ceed->fallback` instead of one per operator?

Reply:
Easy enough to do - it would be consistent per Ceed